acc/kernels/__init__.py

```diff
@@ -45,4 +45,13 @@ def simplepowderdiffraction_scattering_coefficient(neutron):
             return scattering_coefficient(neutron, ucvol, Npeaks, q_v, my_s_v2)
         return simplepowderdiffraction_scatter, simplepowderdiffraction_scattering_coefficient
 
+    def onConstantQEKernel(self, kernel):
+        from .constant_qe import S
+        @cuda.jit(device=True)
+        def constantqe_scatter(threadindex, rng_states, neutron):
+            return S(threadindex, rng_states, neutron, kernel.Q, kernel.E)
```

Comment:
@ckendrick could you please add a line to update the neutron probability with `sigma`, similar to the isotropic kernel?